modules/tinymce/src/core/test/ts/browser/selection/GetSelectionContentTest.ts

```diff
@@ -63,45 +66,55 @@ UnitTest.asynctest('browser.tinymce.selection.GetSelectionContentTest', (success
+      Log.stepsAsStep('TBA', 'Should be empty contents on a caret selection', [
         tinyApis.sSetContent('<p>a</p>'),
         tinyApis.sSetSelection([ 0, 0 ], 0, [ 0, 0 ], 0),
         sAssertGetContent('Should be empty selection on caret', editor, '')
-      ])),
-      Logger.t('Should be text contents on a range selection', GeneralSteps.sequence([
+      ]),
+      Log.stepsAsStep('TBA', 'Should be text contents on a range selection', [
         tinyApis.sSetContent('<p>a</p>'),
         tinyApis.sSetSelection([ 0, 0 ], 0, [ 0, 0 ], 1),
         sAssertGetContent('Should be some content', editor, 'a')
-      ])),
-      Logger.t('Should be text contents provided by override handler', GeneralSteps.sequence([
+      ]),
+      Log.stepsAsStep('TBA', 'Should be text contents provided by override handler', [
         tinyApis.sSetContent('<p>a</p>'),
         tinyApis.sSetSelection([ 0, 0 ], 0, [ 0, 0 ], 1),
         sAssertGetContentOverrideBeforeGetContent('Should be overridden content', editor, 'X')
-      ])),
-      Logger.t(`Should be text contents when editor isn't focused and format is text`, GeneralSteps.sequence([
+      ]),
+      Log.stepsAsStep('TBA', `Should be text contents when editor isn't focused and format is text`, [
         sAddTestDiv,
         tinyApis.sSetContent('<p>ab</p>'),
         tinyApis.sSetSelection([ 0, 0 ], 0, [ 0, 0 ], 2),
         sFocusDiv,
         sAssertGetContent('Should be some content', editor, 'ab', { format: 'text' }),
         sRemoveTestDiv
-      ])),
-      Logger.t('Should be text content with newline', GeneralSteps.sequence([
+      ]),
+      Log.stepsAsStep('TBA', 'Should be text content with newline', [
         tinyApis.sSetContent('<p>ab<br/>cd</p>'),
         tinyApis.sSetSelection([ 0, 0 ], 0, [ 0, 2 ], 2),
         sAssertGetContent('Should be some content', editor, `ab${Env.ie === 11 ? '\r\n' : '\n'}cd`, { format: 'text' })
-      ])),
-      Logger.t('Should be text content with leading visible spaces', GeneralSteps.sequence([
+      ]),
+      Log.stepsAsStep('TBA', 'Should be text content with leading visible spaces', [
         tinyApis.sSetContent('<p>content<em> Leading space</em></p>'),
         tinyApis.sSetSelection([ 0 ], 1, [ 0 ], 2),
         sAssertGetContent('Should be some content', editor, ' Leading space', { format: 'text' })
-      ])),
-      Logger.t('Should be text content with trailing visible spaces', GeneralSteps.sequence([
+      ]),
+      Log.stepsAsStep('TBA', 'Should be text content with trailing visible spaces', [
         tinyApis.sSetContent('<p><em>Trailing space </em>content</p>'),
         tinyApis.sSetSelection([ 0 ], 0, [ 0 ], 1),
         sAssertGetContent('Should be some content', editor, 'Trailing space ', { format: 'text' })
-      ])),
-      Logger.t('Should be text content without non-visible leading/trailing spaces', GeneralSteps.sequence([
+      ]),
+      Log.stepsAsStep('TINY-6448', 'pre blocks should have preserved spaces', [
```

Comment:
Can you make a test with the same spaced out content but using a `<p>` element or similar so that we can verify that the changes only occur for pre blocks?

Reply:
Done